packages/wallets/src/components/Base/ModalStepWrapper/ModalStepWrapper.tsx

```diff
@@ -50,7 +50,7 @@ const ModalStepWrapper: FC<PropsWithChildren<TModalStepWrapperProps>> = ({
     const Footer = () =>
         hasRenderFooter ? (
             <div className='wallets-modal-step-wrapper__footer' data-testid='dt_modal_step_wrapper_footer'>
-                {renderFooter()}
+                <div className='wallets-modal-step-wrapper__footer--content'>{renderFooter()}</div>
```

Comment:
😁 
```suggestion
                <div className='wallets-modal-step-wrapper__footer-content'>{renderFooter()}</div>
```

Reply:
I don't think I agree with you on this 🤔

Reply:
Why not?! 🤔

Reply:
Okay I've been referring this article for my whole life 😂  https://getbem.com/naming/ 
Alright I agree with this